topi/python/topi/nn/util.py

```diff
@@ -0,0 +1,101 @@
+# pylint: disable=invalid-name, unused-variable
+"""NN operator common utilities"""
+from __future__ import absolute_import
+
+def infer_pad(data, data_pad):
+    """Infer the padding from stages in reverse.
+
+    Parameters
+    ----------
+    data : Tensor
+        data stage.
+
+    data_pad : Tensor
+        pad stage.
+
+    Returns
+    -------
+    hpad : int
+        padding size on height
+    wpad : int
+        padding size on width
+    """
+    if data_pad is None:
+        return 0, 0
+    _, _, IH, IW = [x.value for x in data.shape]
+    _, _, TH, TW = [x.value for x in data_pad.shape]
+    hpad = (TH - IH) // 2
```

Comment:
This cannot work if value is symbolic, do substraction in symbolic and then get constant